src/main/java/com/kodilla/ecommercee/repository/ProductRepository.java

```diff
@@ -0,0 +1,21 @@
+package com.kodilla.ecommercee.repository;
+
+import com.kodilla.ecommercee.domain.Product;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import javax.transaction.Transactional;
+import java.util.List;
+import java.util.Optional;
+@Transactional
+@Repository
+public interface ProductRepository extends CrudRepository<Product, Long> {
```

Comment:
We need to make a decision on JpaRepository (suggested in JDP230502-11) and CrudRepository (suggested here and in Jdp230502 15)...

Reply:
Most of the team opted for Crud...